### PR TITLE
Bump to EasyCodingStandard 2.2

### PR DESCRIPTION
**Pull request by TomasVotruba**
Make use of configs to shorten setup in here 

Mailnly PSR2 checkers, before it was bunch of chaotic rules.

See https://www.tomasvotruba.cz/blog/2017/08/07/7-new-features-in-easy-coding-standard-22/

### CODE REVIEW

**Comment by TomasVotruba** [comment]
I've found few bugs in ECS that need to be fixed first. Do not merge yet!

**Comment by TomasVotruba** [comment]
Is ready to merge :+1:

**Comment by jakubzaba** [comment]
Thanks @TomasVotruba. You did a lot of work again. New checkers are great. Made it all clean and simple.

File: app/Modules/Front/Presenters/HomepagePresenter.php [hunk 1]
```diff
@@ -150,9 +155,11 @@ public function handleFollowTag(string $tagCode): void
         $tagsIds = $this->tagModel->getAll()->where('code', $section->tags)->fetchPairs(null, 'id');
 
         $showAbroad = $this->userModel->showAbroadEvents($this->getUser()->getId());
+
         $this->events = $this->eventModel
             ->getAllWithDates($tagsIds, new DateTime, null, null, $showAbroad);
-        $this->followedTags = $section->tags;
+        // what is this for?
```
Comment (jakubzaba) [inline app/Modules/Front/Presenters/HomepagePresenter.php:161]: This is for nothing. Actually, whole `handleFollowTag` and `handleUnfollowTag ` are unfinished work.

**Comment by TomasVotruba** [comment]
Thanks mate, happy to see this help you!

**Pull request merged**